decidim-core/db/seeds.rb

```diff
@@ -17,7 +17,7 @@
     end,
     homepage_image: File.new(File.join(__dir__, "seeds", "homepage_image.jpg")),
     default_locale: I18n.default_locale,
-    available_locales: Decidim.available_locales,
+    available_locales: [:en, :ca, :es],
```

Comment:
Heheh, we replied at the same time :) Sounds good to me, sorry for the trouble :sweat_smile: